### ISSUE TITLE
Hardcoded Templates

### ISSUE THREAD

**Issue by ghost** [Dental-Revenue/Codebase-2-Dev#140]
There are several default pages (leave a review, privacy policy, sitemap, terms of use, a11y) that are contained entirely within the functions.php file. It's my belief that these should be either converted into their own templates, or put in a plugin, so that we can easily edit these pages if need be.
